operations/commit_queue.go

```diff
@@ -117,3 +178,124 @@ func deleteCommitQueueItem(ctx context.Context, client client.Communicator, proj
 
 	return nil
 }
+
+type mergeParams struct {
+	projectID   string
+	ref         string
+	id          string
+	pause       bool
+	description string
+	skipConfirm bool
+	large       bool
+}
+
+func (p *mergeParams) mergeBranch(ctx context.Context, confPath string) error {
+	conf, err := NewClientSettings(confPath)
+	if err != nil {
+		return errors.Wrap(err, "problem loading configuration")
+	}
+
+	ac, _, err := conf.getLegacyClients()
+	if err != nil {
+		return errors.Wrap(err, "problem accessing evergreen service")
+	}
+
+	client := conf.GetRestCommunicator(ctx)
+	if err != nil {
+		return errors.Wrap(err, "problem accessing evergreen service")
+	}
```

Comment:
I'd probably move these kinds of dependency setup operations out of this function. 

also ensure that you don't wrap errors with the same message.

Reply:
done.